### DISCUSSION TITLE
Emqx Java Http Sdk

### DISCUSSION THREAD

**Discussion by gcdd1993** [emqx/emqx#7022] [Show and tell]
I use Java to implement all HTTP API, the repo is https://github.com/gcdd1993/emqx-java-sdk
I hope someone will use and get feedback.

**Comment by hjianbo** [comment]
Excellent repository 👍
One small tip, there may be some minor updates to the API in different EMQX versions, so you need to specify the EMQX version you depend on in your repository.

**Reply by gcdd1993** [reply]
@HJianBo Thanks, i will check the difference between different EMQX versions.
